openpype/hosts/maya/plugins/create/create_rig.py

```diff
@@ -13,12 +13,16 @@ class CreateRig(plugin.Creator):
     label = "Rig"
     family = "rig"
     icon = "wheelchair"
+    write_color_sets = False
+
+    def __init__(self, *args, **kwargs):
+        super(CreateRig, self).__init__(*args, **kwargs)
+        self.data["writeColorSets"] = self.write_color_sets
```

Comment:
This shouldn't make much sense since the rig comes out as maya scene and does not strip any color set data. Animation instances however would extract as Alembic.

Reply:
we are trying to unify colors sets (and other sets) flow and even though currently rig creator can only output Maya scene we plan to add fbx representation (replacing the skeletal mesh) so it will make sense then.

It is not influencing anything currently but we thought it would be good to have it there for future use and consistency.

Reply:
> It is not influencing anything currently but we thought it would be good to have it there for future use and consistency

I see what you're going for - but it'd just confuse the artists into thinking it would do something useful.
Also, most usual "rigs" won't be suitable to do an "fbx rig" export for - and then also automate `animation` instance creation for in maya when loading them again. I'd say `fbxRig` is a family of its own.

Reply:
so we have to decide if we remove the component sets from Rig Creator @antirotor

Reply:
After internal discussion, we decided to keep the sets for Rig Creator there. We understand it is not very clean from the code perspective as these options are actually modifying the animation output, not the rig output itself, but as it does not have its own creator we gave preference to keeping all attributes settings in one place in preferences and not moving it outside just for the rig. 
We think it will be more consistent from the user's point of view.